INTITA/Modules/ProfileSettings/Cells/HeaderSettingsTableViewCell.swift

```diff
@@ -60,4 +58,14 @@ class HeaderSettingsTableViewCell: UITableViewCell, NibCapable {
         mainView.shadowed()
     }
     
+    func setupEditBtn(isTrue: Bool) {
+        
+        if isTrue {
+            editButton.imageView?.image = UIImage(systemName: "checkmarck.circle")
+            editButton.setTitle("  save".localized, for: .normal)
+        } else {
+            editButton.imageView?.image = UIImage(named: "editBtn")
+            editButton.setTitle("  edit".localized, for: .normal)
+        }
```

Comment:
Please use setTitle instead of directly setting imageView.image.

Reply:
It was fixed.